assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135

```diff
@@ -0,0 +1 @@
+
```

Comment:
you need to put logo.png under `assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135`

Reply:
assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135/logo.png
thanks Sarju, pls check if all is good. thx

Reply:
@DrZee3 you haven't added logo.png to `assets/0x6d2FbaDdaB23Fc914aAF6a08f2082eB9d7BC2135` folder. Please go through readme and also check how other logo has been added to repo.